pkg/udev/common.go

```diff
@@ -52,15 +53,20 @@ const (
 	UDEV_SOURCE         = "udev"            // udev source constant
 	UDEV_SYSPATH_PREFIX = "/sys/dev/block/" // udev syspath prefix
 	UDEV_DEVNAME        = "DEVNAME"         // udev attrinute contain disk name given by kernel
+	UDEV_DEVLINKS       = "DEVLINKS"        //udev attrinute contain devlinks of a disk
```

Comment:
should this be UDEV_DEVLINKS or just DEVLINKS?

Reply:
This is a key used by libudev api. This is library defined.

Reply:
We use UDEV as all libudev api key constants's prefix. Thats why it is "UDEV_DEVLINKS"